deps/src/main/java/com/microsoft/azure/sdk/iot/deps/serializer/JobsResponseParser.java

```diff
@@ -20,30 +20,29 @@
  */
 public class JobsResponseParser
 {
-    @Expose(deserialize = false)
-    private static final String DATEFORMAT = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'";
-    @Expose(deserialize = false)
-    private static final String TIMEZONE = "UTC";
-
     // Job identifier
     private static final String JOBID_TAG = "jobId";
     @SerializedName(JOBID_TAG)
+    @SuppressWarnings("unused") // used by reflection during json serialization/deserialization
     private String jobId;
 
     // Required if type is updateTwin or cloudToDeviceMethod.
     // Condition for device query to get devices to execute the job on
     private static final String QUERYCONDITION_TAG = "queryCondition";
     @SerializedName(QUERYCONDITION_TAG)
+    @SuppressWarnings("unused") // used by reflection during json serialization/deserialization
     private String queryCondition;
 
     // Scheduled job start time in UTC.
     private static final String CREATETIME_TAG = "createdTime";
     @SerializedName(CREATETIME_TAG)
+    @SuppressWarnings("unused") // used by reflection during json serialization/deserialization
```

Comment:
Just curious how did you validate this is unused?

Reply:
There's some logic below where we attempt to create a date time object from this string, if it was deserialized into